lsif-java/src/main/scala/com/sourcegraph/lsif_java/SemanticdbPrinters.scala

```diff
@@ -37,7 +43,9 @@ object SemanticdbPrinters {
   private def formatOccurrence(
       out: StringBuilder,
       occ: SymbolOccurrence,
-      line: String
+      line: String,
+      symbols: Map[String, SymbolInformation],
```

Comment:
What is the difference between this map and `Symtab`? Would be nice to have only one symtab